packages/libs/react-components/src/components/IconButton/styles.ts

```diff
@@ -0,0 +1,53 @@
+/* eslint @kadena-dev/typedef-var: 0 */
+// TODO: Remove this when this issue is resolved: https://github.com/kadena-community/kadena.js/issues/201
+
+import { styled } from '../../styles';
+
+export const colorVariant = {
+  default: {
+    background: 'transparent',
+  },
+  primary: {
+    background: '$primarySurface',
+    $$svgColor: '$colors$primaryHighContrast',
+  },
+  positive: {
+    background: '$positiveSurface',
+    $$svgColor: '$colors$positiveHighContrast',
+  },
+  negative: {
+    background: '$negativeSurface',
+    $$svgColor: '$colors$negativeHighContrast',
```

Comment:
missing the gray background, used in the Header of the app

Reply:
That's indeed as we have discussed 👍